packages/custom-props-layer/src/TabBody.tsx

```diff
@@ -164,6 +166,22 @@ export const TabBody: React.FC<TabBodyProps> = (props) => {
               routes={routes}
             />
           );
+        if (propType === "enum") {
+          const enumCustomProps = props.treeOptions.dataTypes[
+            propName
+          ] as EnumCustomProp;
+          const options = enumCustomProps.options;
+          return (
+            <Enum
+              {...props}
+              options={options}
+              {...options}
```

Comment:
Why do we have options twice here?

Reply:
Sorry I was testing how data should be passed without modifying files but then realised changing types would be the best approach and forgot to remove it.

Reply:
Done